scripts/exregional_run_enkf.sh

```diff
@@ -160,6 +160,60 @@ mkdir_vrfy -p ${enkfanal_nwges_dir}
 #
 #-----------------------------------------------------------------------
 #
+# Loop through the members, copy over the background and
+#  observer output (diag*ges*) files to the running directory
+#
+#-----------------------------------------------------------------------
+#
+ cp_vrfy ${fixgriddir}/fv3_coupler.res    coupler.res
+ cp_vrfy ${fixgriddir}/fv3_akbk           fv3sar_tile1_akbk.nc
+ cp_vrfy ${fixgriddir}/fv3_grid_spec      fv3sar_tile1_grid_spec.nc
+
+#
+#-----------------------------------------------------------------------
+#
+# Get nlons (NX_RES) and nlats (NY_RES) from  fv3_grid_spec
+#
+#-----------------------------------------------------------------------
+#
+ for imem in  $(seq 1 $nens) ensmean; do
+
+     if [ ${imem} == "ensmean" ]; then
+        memchar="ensmean"
+        memcharv0="ensmean"
+     else
+        memchar="mem"$(printf %04i $imem)
+        memcharv0="mem"$(printf %03i $imem)
+     fi
+     slash_ensmem_subdir=$memchar
+     if [ ${cycle_type} == "spinup" ]; then
+        bkpath=${cycle_dir}/${slash_ensmem_subdir}/fcst_fv3lam_spinup/INPUT
+        observer_nwges_dir="${NWGES_DIR}/${slash_ensmem_subdir}/observer_gsi_spinup"
+     else
+        bkpath=${cycle_dir}/${slash_ensmem_subdir}/fcst_fv3lam/INPUT
+        observer_nwges_dir="${NWGES_DIR}/${slash_ensmem_subdir}/observer_gsi"
+     fi
+
+     ln_vrfy  -snf  ${bkpath}/fv_core.res.tile1.nc         fv3sar_tile1_${memcharv0}_dynvars
+     ln_vrfy  -snf  ${bkpath}/fv_tracer.res.tile1.nc       fv3sar_tile1_${memcharv0}_tracer
+     ln_vrfy  -snf  ${bkpath}/sfc_data.nc                  fv3sar_tile1_${memcharv0}_sfcdata
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy observer outputs (diag*ges*) to the working directory
+#
+#-----------------------------------------------------------------------
+#
+     for diagfile0 in `ls  ${observer_nwges_dir}/diag*ges*`; do
```

Comment:
suggest changing to use $(...):
`$(ls ${observer_nwges_dir}/diag*ges*)`

Reply:
Changes made. Thank you!